### PR TITLE
Add additional custom dimension for relevant result

### PR DESCRIPTION
**Pull request by DilwoarH**
The new custom dimension is Relevant result shown (83). This should be set to 'Yes' if a  search in the finder search box results in a 'relevant result'. This is set at a hit/pageview level, since a user may do several searches in their session.

Ticket:  https://trello.com/c/lwQK2Pi7/336-set-up-a-b-test-for-answer-boxes

### CODE REVIEW

File: app/assets/javascripts/live_search.js [hunk 1]
```diff
@@ -109,15 +109,29 @@
...
   };
 
+  LiveSearch.prototype.getAdditionalCustomDimensionsForPageView = function() {
+    var custom_dimensions = {};
+
+    // Custom dimension for most relevant result shown
+    var $results = $('.finder-results');
+    var $mostRelevantDocumentLink = $results.find('.document__link--top');
+    if ($mostRelevantDocumentLink.length) {
+      custom_dimensions["dimension83"] = { value: "yes" };
```
Comment (vanitabarrett) [inline app/assets/javascripts/live_search.js]: Sorry I'm not familiar with how we're meant to set custom dimensions. Is this meant to be setting `dimension83` to `{value: "yes" }` or should it just be setting it to `"yes"`?

Comment (DilwoarH) [reply app/assets/javascripts/live_search.js]: You're correct, I already found the issue and pushed up 👍

File: app/assets/javascripts/live_search.js [hunk 2]
```diff
@@ -109,15 +109,29 @@
     }
   };
 
+  LiveSearch.prototype.getAdditionalCustomDimensionsForPageView = function() {
```
Comment (vanitabarrett) [inline app/assets/javascripts/live_search.js]: Is it possible to use something like this: https://github.com/alphagov/prototype-performance-platform-alpha/blob/385a021a526b82caf70c75d703391a584c9ba718/govuk_modules/govuk_frontend_toolkit/docs/analytics.md#set-custom-dimensions-before-tracking-pageviews

**Comment by DilwoarH** [comment]
Blocked by https://github.com/alphagov/static/pull/1676

**Comment by DilwoarH** [comment]
> Blocked by [alphagov/static#1676](https://github.com/alphagov/static/pull/1676)

This has been merged

**Comment by DilwoarH** [comment]
@vanitabarrett Spoke to a performance analyst and they said it was fine.

**Pull request merged**